sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/state/StateStoreRDD.scala

```diff
@@ -74,9 +74,14 @@ class StateStoreRDD[T: ClassTag, U: ClassTag](
 
     // If we're in continuous processing mode, we should get the store version for the current
     // epoch rather than the one at planning time.
-    val currentVersion = EpochTracker.getCurrentEpoch match {
-      case None => storeVersion
-      case Some(value) => value
+    val isContinuous = Option(ctxt.getLocalProperty(ContinuousExecution.IS_CONTINUOUS_PROCESSING))
```

Comment:
Just simple `toBoolean` here is OK？Cause you set default value both MicroBatch and Continuous side.

Reply:
I think I'd rather keep it as is to be more resilient for the future.